x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/stats/FeatureMetric.java

```diff
@@ -32,26 +33,25 @@ public enum FeatureMetric {
     /**
      * The order of these enum values is important, do not change it.
      * For any new values added to it, they should go at the end of the list.
```

Comment:
Relying on enum ordering is brittle.  We should add an ordering parameter to the enum and use that for whatever we're currently depending on order for.  I'd be fine with doing that in a follow up though.